### PR TITLE
Second %pre should be %post

### PR DESCRIPTION
**Pull request by dlanderson**
Fixes https://github.com/jordansissel/fpm/issues/803

### CODE REVIEW

**Comment by djhaskin987** [comment]
+1. Thanks for fixing this, my bad! :)

File: templates/rpm.erb [hunk 1]
```diff
@@ -139,7 +139,7 @@ then
 fi
 <%   end -%>
 <%   if script?(:after_upgrade) or script?(:after_install) -%>
-%pre
+%post
```
Comment (jordansissel) [inline templates/rpm.erb]: doh! Good catch.

**Comment by jordansissel** [comment]
Eyeballed; lgtm! I didn't test yet, but that's ok. :)

**Comment by djhaskin987** [comment]
I tested this :+1:

**Comment by jordansissel** [comment]
:heart:

**Pull request merged**